remoteworkitem/remoteworkitem_test.go

```diff
@@ -43,14 +59,15 @@ func TestWorkItemMapping(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if workItem.Fields["system.title"] == nil {
+	if !assert.NotNil(t, workItem.Fields["system.title"]) {
```

Comment:
Thanks @kwk ,
The reason why I kept the 
```
     t.Error("system.title not mapped")
``` 
is because I wanted to preserve what exactly is the key corresponding to which the mapping was absent. 

Not having this line only gave me the "expected" and "actual" values which wasn't really helpful in figuring out what went wrong.